ostelco-ios-client/ViewControllers/Home/SetupApplePayViewController.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  SetupApplePayViewController.swift
+//  ostelco-ios-client
+//
+//  Created by Prasanth Ullattil on 05/09/2019.
+//  Copyright © 2019 mac. All rights reserved.
+//
+
+import OstelcoStyles
+import PassKit
+import PromiseKit
+import Stripe
+import UIKit
+
+class SetupApplePayViewController: ApplePayViewController {
+
+    @IBOutlet private var buttonContainer: UIView!
+    @IBOutlet private var explanatoryCopyLabel: BodyTextLabel!
+
+    var paymentButton: PKPaymentButton?
+    var membership: Product?
+
+    lazy var linkableCopy: LinkableText = {
```

Comment:
Just for the record: there's no need to store this stuff anymore. The delegate method now informs which link was tapped, so we don't need to save it to match against.

Reply:
Can you suggest what I do here. I was just copying the structure in the "BecomeAMemberViewController"

Reply:
I mean you can create and assign the text directly. There's no need to make a property for this text.

Reply:
Remove `linkableCopy` property